packages/holocron/__tests__/HolocronErrorBoundary.spec.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { mount } from 'enzyme';
+import HolocronErrorBoundary, { withHolocronErrorBoundary } from '../src/HolocronErrorBoundary';
+
+describe('HolocronErrorBoundary', () => {
+  let consoleErrorSpy;
+  let mockError;
+  let CharacterWelcome;
+  let Character;
+
+  const { NODE_ENV } = process.env;
+
+  beforeAll(() => {
+    consoleErrorSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
+    mockError = new Error(
+      'These are not the droids you are looking for.'
+    );
+
+    // eslint-disable-next-line react/prop-types
+    Character = ({ name }) => {
```

Comment:
add prop `throwError` or something so that is explicit when it should throw or not

Reply:
Taken care of in 3649138